docs/platforms/dotnet/migration/index.mdx

```diff
@@ -6,6 +6,138 @@ sidebar_order: 8000
+- `StackFrame`'s `ImageAddress`, `InstructionAddress`, and `FunctionId` changed to `long?`.
+- `DebugImage.ImageAddress` changed to `long?`.
+- Contexts now inherit from `IDictionary` rather than `ConcurrentDictionary`. The specific dictionary being used is an implementation detail.
+- The method used to configure a Sentry Sink for Serilog now has an additional overload. Calling `WriteTo.Sentry()` with no arguments will no longer attempt to initialize the SDK (it has optional arguments to configure the behavior of the Sink only). If you want to initialize Sentry at the same time you configure the Sentry Sink then you will need to use the overload of this method that accepts a DSN as the first parameter (e.g. `WriteTo.Sentry("https://d4d82fc1c2c4032a83f3a29aa3a3aff@fake-sentry.io:65535/2147483647")`).
+
+#### Removed APIs
+
+- SentrySinkExtensions.ConfigureSentrySerilogOptions is now internal. If you were using this method, please use one of the `SentrySinkExtensions.Sentry` extension methods instead.
+- A number of `[Obsolete]` options have been removed
+  - `BeforeSend` - use `SetBeforeSend` instead.
+  - `BeforeSendTransaction` - use `SetBeforeSendTransaction` instead.
+  - `BeforeBreadcrumb` - use `SetBeforeBreadcrumb` instead.
+  - `CreateHttpClientHandler` - use `CreateHttpMessageHandler` instead.
+  - `ReportAssemblies` - use `ReportAssembliesMode` instead.
+  - `KeepAggregateException` - this property is no longer used and has no replacement.
+  - `DisableTaskUnobservedTaskExceptionCapture` method has been renamed to `DisableUnobservedTaskExceptionCapture`.
+  - `DebugDiagnosticLogger` - use `TraceDiagnosticLogger` instead.
+- A number of iOS/Android-specific `[Obsolete]` options have been removed
+  - `Distribution` - use `SentryOptions.Distribution` instead.
+  - `EnableAutoPerformanceTracking` - use `SetBeforeSendTransaction` instead.
+  - `EnableCoreDataTracking` - use `EnableCoreDataTracing` instead.
+  - `EnableFileIOTracking` - use `EnableFileIOTracing` instead.
+  - `EnableOutOfMemoryTracking` - use `EnableWatchdogTerminationTracking` instead.
+  - `EnableUIViewControllerTracking` - use `EnableUIViewControllerTracing` instead.
+  - `StitchAsyncCode` - no longer available.
+  - `ProfilingTracesInterval` - no longer available.
+  - `ProfilingEnabled` - use `ProfilesSampleRate` instead.
+- Obsolete `SystemClock` constructor removed, use `SystemClock.Clock` instead.
+- Obsolete `Runtime.Clone()` removed, this shouldn't have been public in the past and has no replacement.
+- Obsolete `SentryException.Data` removed, use `SentryException.Mechanism.Data` instead.
+- Obsolete `AssemblyExtensions` removed, this shouldn't have been public in the past and has no replacement.
+- Obsolete `SentryDatabaseLogging.UseBreadcrumbs()` removed, it is called automatically and has no replacement.
+- Obsolete `Scope.GetSpan()` removed, use `Span` property instead.
+- Obsolete `IUserFactory` removed, use `ISentryUserFactory` instead.
```

Comment:
```suggestion
- Obsolete `SystemClock` constructor removed. Use `SystemClock.Clock` instead.
- Obsolete `Runtime.Clone()` removed. This shouldn't have been public in the past and has no replacement.
- Obsolete `SentryException.Data` removed. Use `SentryException.Mechanism.Data` instead.
- Obsolete `AssemblyExtensions` removed. This shouldn't have been public in the past and has no replacement.
- Obsolete `SentryDatabaseLogging.UseBreadcrumbs()` removed. It is called automatically and has no replacement.
- Obsolete `Scope.GetSpan()` removed. Use `Span` property instead.
- Obsolete `IUserFactory` removed. Use `ISentryUserFactory` instead.
```

Reply:
Is there a good way to group these and the bullets below it together somehow? like how the iOS/Android options are together